models/model.js

```diff
@@ -5,3 +5,17 @@ exports.topicsModel = () => {
     return result.rows;
   });
 };
+
+exports.articleIdModel = (article_id) => {
```

Comment:
Again the name of this function should be representative of what it does. Doesn't need to include where it sits in MVC chain. A name like fetchArticleById or selectArticleId for this one.